### PR TITLE
Add system table upgrade

### PR DESCRIPTION
**Pull request by qingxinhome**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #11116

## What this PR does / why we need it:
Add system tables and upgrade system views

### CODE REVIEW

File: pkg/util/export/table/table.go [hunk 1]
```diff
@@ -285,6 +291,9 @@ type Table struct {
 	name2ColumnIdx map[string]int
 	// accessIdx used in Row
 	accountIdx int
+
+	CreateTableSql string
```
Comment (xzxiong) [inline pkg/util/export/table/table.go:295]: 麻烦被备注一下，这 2个字段的使用场景，以及与 `table.ToCreateSql` 的区别。

File: pkg/util/export/table/table.go [hunk 2]
```diff
@@ -51,6 +51,12 @@ type ColType int
 const (
 	TSkip ColType = iota
 	TDatetime
+	TUint8
```
Comment (xzxiong) [inline pkg/util/export/table/table.go:54]: 请问这些类型真的需要么？

File: pkg/util/export/table/table.go (hunk 1 above)
Comment (qingxinhome) [reply pkg/util/export/table/table.go:295]: I think it's not necessary

File: pkg/util/export/table/table.go (hunk 2 above)
Comment (qingxinhome) [reply pkg/util/export/table/table.go:54]: yes， you are right， I think it's not necessary

File: pkg/util/export/table/table.go (hunk 1 above)
Comment (qingxinhome) [reply pkg/util/export/table/table.go:295]: ok

**Review by reusee — APPROVED** [review]

**Review by gavinyue — APPROVED** [review]

**Review by nnsgmsone — APPROVED** [review]

**Review by xzxiong — APPROVED** [review]

**Pull request merged**